git_pw/patch.py

```diff
@@ -329,6 +329,24 @@ def update_cmd(patch_ids, commit_ref, state, delegate, archived, fmt):
     is_flag=True,
     help='Include patches that are archived.',
 )
+@click.option(
+    '--since',
+    'since',
+    metavar='SINCE',
```

Comment:
Can you make this a `click.DateTime` format? Ditto for the rest of these

API docs: https://click.palletsprojects.com/en/8.0.x/parameters/
Random blog: https://www.markhneedham.com/blog/2019/07/29/python-click-date-parameter-type/